advocacy_docs/edb-postgres-ai/analytics/external_tables.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Querying Delta tables in S3-compatible object storage
+navTitle: External Tables
+description: Access and Query data in S3-compatible object storage using External Tables
+deepToC: true
+---
+
+## Overview
+
+External tables allow you to access and query data stored in S3-compatible object storage using SQL. You can create an external table that references data in S3-compatible object storage and query the data using standard SQL commands.
+
+## Prerequisites
+
+An EDB Postgres AI account and a Lakehouse node.
+
+## Creating an External Storage Location
+
+The first step is to create an external storage location which references S3-compatible object storage where your data resides. This is done by using SQL to execute the `pgaa.create_storage_location` function. `pgaa` is the name of the extension and namespace that provides the functionality to query external storage locations.
```

Comment:
BTW this will change when we start using the pgfs extension to manage external storage locations for both PGAA and AIDB (https://docs.google.com/document/d/1Td-57HLPngUAqLI3fz31n6khSg4EEqeUGY-nXHvqrYw/edit?tab=t.0#heading=h.13bitxjx3mm6), there the user will run something like

```
CREATE SERVER my_s3_bucket
    FOREIGN DATA WRAPPER pgfs_s3
    OPTIONS ( url 's3://bucket', msl_id '', option_a '...' )
...
CREATE USER MAPPING FOR PUBLIC
    SERVER my_s3_bucket
    OPTIONS ( aws_access_key 'xyz', aws_secret_key 'xyz', vault_key '/pgfs/bucket' )
```